tictactoe.js

```diff
@@ -0,0 +1,136 @@
+
+player= "X";
+space = document.getElementsByClassName("space");
+
+
+// var boardCheck;
+
+// var cellValue = function cells(key) {
+//     switch(key) {
+//         case 'a': return "";
+//         case 'b': return "";
+//         case 'c': return "";
+//         case 'd': return "";
+//         case 'e': return "";
+//         case 'f': return "";
+//         case 'g': return "";
+//         case 'h': return "";
+//         case 'i': return "";
+//         default : return "";
+//     }
+// }
+
+
+//-------------------------------------------------------------------
+function clickSpace() {
+    if (this.id == "a") {
```

Comment:
This is a pretty neat approach to this. You could also use a switch statement here if you wanted.